### DISCUSSION TITLE
Master Server Not showing alerts from other sensors

### DISCUSSION THREAD

**Discussion by mamba143** [Security-Onion-Solutions/securityonion#3212] [Unsupported Versions] [answered]
Master Server is only showing alerts from Master server but not sensor servers. Sensor servers are sending logs to the master and are visible via hunt and kibana but not alerts

**Comment by weslambert** [comment]
Could it be that you are just not seeing traffic from those hosts that should be generating alerts?

**Reply by mamba143** [reply]
No, I can confirm that I am receiving data from the hosts. HUNT, KIBANA grafana works, however the alerts is only showing information form the the master

**Comment by mamba143** [comment, marked as answer]
I figured out the issue, I did not realize the server did not have internet connection. So I believe it could not have downloaded the latest signatures for identification.
